spread/project.go

```diff
@@ -985,6 +986,19 @@ func (p *Project) Jobs(options *Options) ([]*Job, error) {
 		if !manualTasks && job.Task.Manual {
 			continue
 		}
+
+		// Check the tag in the jon in case a tag is specified in the options
+		if options.Tag != "" {
+			tagMatch := false
+			for tag := range job.Task.Tags {
+				if options.Tag == job.Task.Tags[tag] {
+					tagMatch = true
+				}
+			}
+			if !tagMatch {
```

Comment:
but, actually, note there's a `contains()` defined in `lxd.go` so drop the whole loop and if and instead simply do
```go
if options.Tag != "" && !contains(job.Task.Tags, options.Tag) {
    continue
}
```

Reply:
@chipaca in this case it is not importing the lxd.go file, should I create a utils.go and move that function there instead?

Reply:
@sergiocazzolato it's in the same package so it's already "there". I don't think moving it to utils.go would be useful (it's very small).

Reply:
Ah, nice, thanks for the answer!